.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-app.js
\ No newline at end of file
+carousel.app.js
```

Comment:
Add your eslintrc.js file to your gitignore. It is a config file that doesn't need to be pushed to your repo

Reply:
Made the changes, thanks.